tasks/main.yml

```
@@ -6,6 +6,10 @@
     shell: opkg list-installed | grep dnsmasq-full | awk '{print $3}'
     register: dnsmasqfull_version
 
+  - name: Check confdir option
+    shell: uci get dhcp.@dnsmasq[0].confdir
+    register: dnsmasq_confdir
```

```suggestion
    register: dnsmasq_confdir
    ignore_errors: true
```
т.к. может не быть этого параметра будет ошибка